admin-client/src/components/SiteMetadata.svelte

```diff
@@ -3,7 +3,6 @@
   import ExternalLink from './ExternalLink.svelte';
```

Comment:
Sooo, i removed this bc I wasn't including the users in the query and it was causing problems. Trying to think of why we would want/need to see the users...